specs/light_client/merkle_proofs.md

```diff
@@ -6,17 +6,52 @@
 <!-- TOC -->
 
 - [Merkle proof formats](#merkle-proof-formats)
-   - [Table of contents](#table-of-contents)
-   - [Constants](#constants)
-   - [Generalized Merkle tree index](#generalized-merkle-tree-index)
-   - [SSZ object to index](#ssz-object-to-index)
-   - [Merkle multiproofs](#merkle-multiproofs)
-   - [MerklePartial](#merklepartial)
-       - [`SSZMerklePartial`](#sszmerklepartial)
-       - [Proofs for execution](#proofs-for-execution)
+    - [Table of contents](#table-of-contents)
+    - [Custom types](#custom-types)
+    - [Helper functions](#helper-functions)
+    - [Generalized Merkle tree index](#generalized-merkle-tree-index)
+    - [SSZ object to index](#ssz-object-to-index)
+        - [Helpers for generalized indices](#helpers-for-generalized-indices)
+            - [`concat_generalized_indices`](#concat_generalized_indices)
+            - [`get_generalized_index_length`](#get_generalized_index_length)
+            - [`get_generalized_index_bit`](#get_generalized_index_bit)
+            - [`generalized_index_sibling`](#generalized_index_sibling)
+            - [`generalized_index_child`](#generalized_index_child)
+            - [`generalized_index_parent`](#generalized_index_parent)
+    - [Merkle multiproofs](#merkle-multiproofs)
 
 <!-- /TOC -->
 
+## Custom types
+
+We define the following Python custom types for type hinting and readability:
+
+| - | - | - |
+| `GeneralizedIndex` | `uint64` | the index of a node in a binary Merkle tree |
+
+## Helper functions
+
+```python
+def get_next_power_of_two(x: int) -> int:
+    """
+    Get next power of 2 >= the input.
+    """
+    if x <= 2:
+        return x
+    elif x % 2 == 0:
+        return 2 * get_next_power_of_two(x // 2)
+    else:
+        return 2 * get_next_power_of_two((x + 1) // 2)
```

Comment:
Why not just replace both of the last two sections with `return 2 * get_next_power_of_two((x + 1) // 2)`?